driver-core/src/main/java/com/datastax/driver/core/querybuilder/QueryBuilder.java

```diff
@@ -351,6 +351,19 @@ public static Clause in(String name, Object... values) {
      * @param values the values
      * @return the corresponding where clause.
      */
+    public static Clause in(String name, Collection<?> values) {
```

Comment:
As a side note, I wonder why the wildcards are necessary in those methods parameters for our use case (they are in many other methods parameters of the query builder). Was looking around and couldn't find a reason why we can't just put `Collection` instead of `Collection<?>`, but maybe it's a Java < 8 thing, or a best practice / usual things to do that I'm not aware of (happy to be explained then).

Reply:
It's more of a best practice than anything else, I think they are functionally the same.  I believe some IDEs may warn when using collections with no generic type.

Reply:
A long time ago, before Java had generics, all collections were raw. You would receive a collection in a method and cast the elements out of it. You could pass a collection of the wrong type and it would only fail at runtime. It sucked.
Nowadays if you use `Collection` in your API, the client is left wondering: did you live in a cave since Java 1.4 or is it an oversight? Will the method try to cast elements to a particular type?
If you use `Collection<?>`, you document the fact that any element type works.
Admittedly this is a subtle difference and it's all the same after erasure, but I think clear contracts matter.

Reply:
> A long time ago,

in a galaxy far far away...

> it's all the same after erasure

Yes, especially because in the rest of the codebase of the query builder we just declare the objects that we get from this collection as `Object`s. 

Although, the idea of expressing explicitly that you accept objects from any type, instead of leaving it empty and leaving the uncertainty of not knowing whether the fact that there's no type means that "any type" would work, or it means that it expects one specific type but didn't mention it makes sense. Thanks for the clarifications.